test/integration/features/cmd-profile.feature

```diff
@@ -75,28 +75,26 @@ Feature: Profile commands
   Scenario: As user, I can apply independent settings in profile 'foo'
      When executing "minishift profile set foo" succeeds
      Then profile "foo" should be the active profile
-     When executing "minishift --profile minishift config get memory" succeeds
-     Then stdout should match "<nil>"
+      And executing "minishift --profile minishift config set cpus 4" succeeds
       And executing "minishift --profile minishift addons list" succeeds
       And stdout should match "registry-route\s*: disabled\s*P\(0\)"
       And executing "minishift addons enable registry-route" succeeds
       And executing "minishift config set memory 5120" succeeds
       And executing "minishift config get memory" succeeds
       And stdout should match "5120"
+      And image caching is disabled
      When executing "minishift start" succeeds
-     Then stdout should contain
-      """
-      Add-on 'registry-route' created docker-registry route.
-      """
+     Then stdout should match "vCPUs\s*:\s*2"
```

Comment:
@amitkrout how come vCPUs is 2 here instead 4?

Reply:
I am not adding the any CPUs to foo profile. I added the CPUs to default minishift profile, so it wont reflect in profile foo